src/lib/UniversalDiscovery/Event/Subscriber/RelationListAllowedContentTypes.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\UniversalDiscovery\Event\Subscriber;
+
+use EzSystems\EzPlatformAdminUi\UniversalDiscovery\Event\ConfigResolveEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class RelationListAllowedContentTypes implements EventSubscriberInterface
```

Comment:
I could be wrong but `ObjectRelationStartingLocationId` seems to have similar role but for the `starting_location_id` parameter. Should we keep convention and name this listener `ObjectRelationAllowedContentTypes` and also limit the scope of it  same as in the following example: https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/lib/UniversalDiscovery/Event/Subscriber/ObjectRelationStartingLocationId.php#L37-L43 ?

Reply:
@adamwojs when it comes to naming, I could probably agree, yet I didn't notice any convention, e.g.: https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/lib/UniversalDiscovery/Event/Subscriber/RichTextAllowedLanguages.php. When it comes to the condition: wouldn't it be less readable? Introduced Subscriber applies only to allowed CT's for ezobjectrelationlist so checking them should be enough to make it work properly. Or?